app/src/main/java/com/es/findsoccerplayers/RegisterActivity.java

```diff
@@ -94,24 +86,24 @@ public void onClick(View v) {
                     edConfEmail.setError("Campo obbligatorio");
                     return;
                 }
+                if(TextUtils.isEmpty(psw)){
+                    edPsw.setError("Campo obbligatorio");
+                    return;
+                }
+                if(TextUtils.isEmpty(confPsw)){
+                    edConfPsw.setError("Campo obbligatorio");
+                    return;
+                }
                 if(!email.equals(confEmail)){
                     edEmail.setError("Le email non coincidono");
                     return;
                 }else{
                     edEmail.setError(null);
                 }
-                if(TextUtils.isEmpty(psw)){
-                    edPsw.setError("Campo obbligatorio");
-                    return;
-                }
                 if(psw.length() < 6) {
                     edPsw.setError("La password deve avere minimo 6 caratteri");
                     return;
                 }
-                if(TextUtils.isEmpty(confPsw)){
-                    edConfPsw.setError("Campo obbligatorio");
-                    return;
-                }
                 if(!psw.equals(confPsw)){
                     edPsw.setError("Le password non coincidono");
                     return;
```

Comment:
Non hai tolto i `return` in modo da mostrare all'utente tutti gli errori di compilazione dei campi